scripts/logger_pc_app/README.md

```diff
@@ -0,0 +1,123 @@
+# ZephyrLogParser
+
+Small application written in Python 3, which purpose is to collect data bytes from target device via backends (e.g GPIO parallel or UART) and parse them.
+It allows faster logging rate, due to fact that there will be no need for usage of costly string-parsing functions on target device side. All human-readable strings are constructed on PC host.
```

Comment:
Why bitbang GPIO's instead of using a serial peripheral?

Wouldn't that be much faster?

Reply:
Because you may use all HW peripherals for something else on your board. The few GPIOs is easier to find that additional peripherals.

Reply:
Then that should be in the motivation.

Cause this does not look fast. But it will be unlikely to conflict with the peripheral resources used by the app.

The cleanest solution would be to know what HW resources are used by the app and then use what is unused, but I guess that is out-of-scope. I'm sceptical there is a use-case where all HW peripherals are used simultaneously.